jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/AbstractRuntimeManager.java

```diff
@@ -125,6 +125,18 @@ public AbstractRuntimeManager(RuntimeEnvironment environment, String identifier)
         }
         this.executionErrorManager = new ExecutionErrorManagerImpl(storage);
         ((SimpleRuntimeEnvironment)environment).getEnvironmentTemplate().set(EnvironmentName.EXEC_ERROR_MANAGER, executionErrorManager);
+        if ((this instanceof SingletonRuntimeManager) && (environment instanceof SchedulerProvider)) {
+            GlobalSchedulerService schedulerService = ((SchedulerProvider) environment).getSchedulerService();
+            Class<?> clazz;
+            try {
+                clazz = Class.forName("org.jbpm.services.ejb.timer.EjbSchedulerService");
```

Comment:
Cannot this also happen with the Quartz scheduler?

Reply:
no, as quartz does not control transaction so it's not an issue